packages/core/src/components/tree/test/tree.ct.ts

```diff
@@ -61,10 +61,16 @@ const defaultModel = {
   },
 };
 
-const initializeTree = async (mount: Mount, page: Page) => {
+const initializeTree = async (
+  mount: Mount,
+  page: Page,
+  toggleOnItemClick?: boolean
+) => {
   await mount(`
       <div style=" height: 20rem; width: 100%;">
-        <ix-tree root="root"></ix-tree>
+        <ix-tree root="root" ${
+          toggleOnItemClick ? 'toggle-on-item-click' : ''
+        }></ix-tree>
```

Comment:
We should use tree.evaluate to set the attribute instead of adding a param here.